### DISCUSSION TITLE
Share a single PVC between multiple prometheus pods

### DISCUSSION THREAD

**Discussion by kaushiklala** [prometheus-operator/kube-prometheus#1636] [Q&A]
Hi,

How do you use a single PVC across multiple Prometheus replicas.

I am using a manually created PV, I have configured the jsonnet to use the PV, using the selector.  This works if there is 1 replica.  If there are multiple replicas, it is expected that there is an equal number of PVs.  Since the Prometheus operator generate the PVC and each PVC has a unique name.

Is there a way to configure the jsonnet so that a single PVC can be used by multiple replicas, i.e how to configure the jsonnet such that the operator generates only one PVC that will be share by all the replicas.

**Comment by metalmatze** [comment]
If I'm not mistaken for that feature you would need ReadWriteMany for the PVC and that's something that the operator won't support.
https://github.com/prometheus-operator/prometheus-operator/issues/2535
